lib/DDG/Spice/Cryptocurrency.pm

```diff
@@ -54,24 +55,40 @@ my @availableLocalCurrencies = (
     'eur',
     'jpy',
     'rur',
+    'rub',
     'uah'
 );
 
+foreach my $currency (@currencies){
+    chomp($currency);
+    my @currency = split(/,/,$currency);
+
+    if(!grep(/^$currency[0]$/, @availableLocalCurrencies) || !grep(/^$currency[0]$/, @excludedCurrencies)) {
```

Comment:
more Perl-y approach: `push(...) unless ( grep ... || grep ...)`

or just use an `unless` block, instead of `if(!...`